### PR TITLE
Add -nm option to not copy mappings

### PR DESCRIPTION
**Pull request by wader**
Not sure if this is the correct way of doing it but i'm using a elastichsearch with "dynamic_templates" configured and copying the mappings seems to override those mappings

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -25,11 +25,12 @@ and mapping copied.
 
 Usage:
 
-  ./es-reindex.rb [-r] [-f <frame>] [source_url/]<index> [destination_url/]<index>
```
Comment (JustinAiken) [inline README.markdown:28]: Ooh good catch!

**Comment by JustinAiken** [comment]
Looks good to me :+1: 

Look okay to you, @benlangfeld?

**Comment by benlangfeld** [comment]
> Look okay to you, @benlangfeld?

`¯\_(ツ)_/¯`

File: README.markdown (hunk 1 above)
Comment (wader) [inline README.markdown:28]: Was more adding even more options would make the line so long :)

**Comment by JustinAiken** [comment]
Out as [0.3.3](https://github.com/mojolingo/es-reindex/releases/tag/v0.3.3)

**Pull request merged**